.github/ISSUE_TEMPLATE/out_of_date_package_report.yml

```diff
@@ -0,0 +1,53 @@
+name: Out-of-date package report
+description: A package has not been updated to the latest version.
+labels: '9.needs: package (update)'
+
+body:
+  - type: input
+    attributes:
+      label: Package name
+      description: Name of the package which is out-of-date.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Current version
+      description: |
+        Check the [master branch](https://github.com/NixOS/nixpkgs) to find the current version.
+        1. Use the "Go to file" functionality on GitHub to find the package.
+        2. Then, go to the history for this package.
+        3. Find the latest "package_name: old_version -> new_version" commit.
+        4. The "new_version" is the current version of the package.
+      placeholder: X.Y.Z
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Desired version
+      description: What version does the package need to be updated to?
+      placeholder: X.Y.Z
+    validations:
+      required: true
```

Comment:
This is how it was before, so don't consider this a blocker but I'd love to not have these fields at all.

Arch Linux has a beautifully simple "flag out-of-date" button that simply tells the maintainer: "Hey, there's a new version. Do something about it."  
No forms required.

If you think about it, the maintainer will have to go to the source, copy version numbers etc. themselves anyways, so it's not like they *need* to know precisely which version it was before and which it should be.  
I don't think this bureaucracy is useful.